<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="listViewComs"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Адрес</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="addrBox">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listViewMsg"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Atternuator 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="atten1">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Attenuator 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="atten2">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="Module_OK">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Module</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="PSU_OK">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>PSU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ROM_OK">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>ROM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Temp_OK">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>TempSensor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Volt_OK">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>VoltSensor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Atten1_OK">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>atten1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Atten2_OK">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>atten2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Res">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Res</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Major Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MajorV">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Minor Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MinorV">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TempStat">
           <property name="text">
            <string>Temp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VoltStat">
           <property name="text">
            <string>Voltage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Обновить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>78</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
